src/coreclr/scripts/.pylintrc

```diff
@@ -0,0 +1,31 @@
+# Settings for pylint
+
+[MESSAGES CONTROL]
+
+# Disable the message, report, category or checker with the given id(s). You
```

Comment:
With this file, does this show errors/warnings in VS Code? I might have to run this on the new file `superpmi-setup.py` that I am adding.